rpc/wrtc_call_queue_mongodb.go

```diff
@@ -677,14 +673,14 @@ func (queue *mongoDBWebRTCCallQueue) processNextSubscriptionEvent(next mongoutil
 func (queue *mongoDBWebRTCCallQueue) subscriptionManager(currentCS <-chan mongoutils.ChangeEventResult) {
 	var waitForNextCS bool
 	for {
-		if queue.cancelCtx.Err() != nil {
+		if queue.activeStoppableWorkers.Context().Err() != nil {
```

Comment:
As per above, if we add in the `ctx` to this call, we can swap out accessing `activeStoppableWorkers` here.